src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -530,40 +535,44 @@ async function ProcessHTTPS(frb: FunctionsRuntimeBundle, trigger: EmulatedTrigge
 
 async function ProcessBackground(
   frb: FunctionsRuntimeBundle,
-  stubbedAdminModule: typeof admin,
   trigger: EmulatedTrigger
 ): Promise<void> {
   const { Change } = require("firebase-functions");
   new EmulatorLog("SYSTEM", "runtime-status", "ready").log();
 
+  if (!app) {
```

Comment:
Should this also be in `ProcessHTTPS`?  Maybe it should be one level higher and just be in `main` before we invoke anything?